dashboards/sql/household_income_distribution.sql

```
@@ -0,0 +1,35 @@
+WITH filtered AS (
+    SELECT monthly_income * 12 AS annual_income
+    FROM analytics.mart_screener_data
+    WHERE 1=1 [[AND {{partner}}]]
```

SQLFluff (4.0.4)

[error] Expected single whitespace between numeric literal and raw comparison operator '='.

(LT01)